src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -133,4 +133,6 @@ public class KNNConstants {
     // Please refer this github issue for more details for choosing this value:
     // https://github.com/opensearch-project/k-NN/issues/1049#issuecomment-1694741092
     public static int MAX_DISTANCE_COMPUTATIONS = 2048000;
+
+    public static float LUCENE_RADIAL_SEARCH_TRAVERSAL_SIMILARITY_RATIO = 0.95f;
```

Comment:
Should we include default in the constant name somewhere?

Reply:
Added